subprojects/packagefiles/icu/source/i18n/meson.build

```diff
@@ -1,255 +1,5 @@
-  'number_longnames.cpp',
-  'number_mapper.cpp',
-  'number_modifiers.cpp',
-  'number_multiplier.cpp',
-  'number_notation.cpp',
-  'number_output.cpp',
-  'number_padding.cpp',
-  'number_patternmodifier.cpp',
-  'number_patternstring.cpp',
-  'number_rounding.cpp',
-  'number_scientific.cpp',
-  'number_skeletons.cpp',
-  'number_symbolswrapper.cpp',
-  'number_usageprefs.cpp',
-  'number_utils.cpp',
-  'numfmt.cpp',
-  'numparse_affixes.cpp',
-  'numparse_compositions.cpp',
-  'numparse_currency.cpp',
-  'numparse_decimal.cpp',
-  'numparse_impl.cpp',
-  'numparse_parsednumber.cpp',
-  'numparse_scientific.cpp',
-  'numparse_symbols.cpp',
-  'numparse_validators.cpp',
-  'numrange_capi.cpp',
-  'numrange_fluent.cpp',
-  'numrange_impl.cpp',
-  'numsys.cpp',
-  'olsontz.cpp',
-  'persncal.cpp',
-  'pluralranges.cpp',
-  'plurfmt.cpp',
-  'plurrule.cpp',
-  'quant.cpp',
-  'quantityformatter.cpp',
-  'rbnf.cpp',
-  'rbt.cpp',
-  'rbt_data.cpp',
-  'rbt_pars.cpp',
-  'rbt_rule.cpp',
-  'rbt_set.cpp',
-  'rbtz.cpp',
-  'regexcmp.cpp',
-  'regeximp.cpp',
-  'regexst.cpp',
-  'regextxt.cpp',
-  'region.cpp',
-  'reldatefmt.cpp',
-  'reldtfmt.cpp',
-  'rematch.cpp',
-  'remtrans.cpp',
-  'repattrn.cpp',
-  'rulebasedcollator.cpp',
-  'scientificnumberformatter.cpp',
-  'scriptset.cpp',
-  'search.cpp',
-  'selfmt.cpp',
-  'sharedbreakiterator.cpp',
-  'simpletz.cpp',
-  'smpdtfmt.cpp',
-  'smpdtfst.cpp',
-  'sortkey.cpp',
-  'standardplural.cpp',
-  'string_segment.cpp',
-  'strmatch.cpp',
-  'strrepl.cpp',
-  'stsearch.cpp',
-  'taiwncal.cpp',
-  'timezone.cpp',
-  'titletrn.cpp',
-  'tmunit.cpp',
-  'tmutamt.cpp',
-  'tmutfmt.cpp',
-  'tolowtrn.cpp',
-  'toupptrn.cpp',
-  'translit.cpp',
-  'transreg.cpp',
-  'tridpars.cpp',
-  'tzfmt.cpp',
-  'tzgnames.cpp',
-  'tznames.cpp',
-  'tznames_impl.cpp',
-  'tzrule.cpp',
-  'tztrans.cpp',
-  'ucal.cpp',
-  'ucln_in.cpp',
-  'ucol.cpp',
-  'ucol_res.cpp',
-  'ucol_sit.cpp',
-  'ucoleitr.cpp',
-  'ucsdet.cpp',
-  'udat.cpp',
-  'udateintervalformat.cpp',
-  'udatpg.cpp',
-  'ufieldpositer.cpp',
-  'uitercollationiterator.cpp',
-  'ulistformatter.cpp',
-  'ulocdata.cpp',
-  'umsg.cpp',
-  'unesctrn.cpp',
-  'uni2name.cpp',
-  'units_complexconverter.cpp',
-  'units_converter.cpp',
-  'units_data.cpp',
-  'units_router.cpp',
-  'unum.cpp',
-  'unumsys.cpp',
-  'upluralrules.cpp',
-  'uregex.cpp',
-  'uregexc.cpp',
-  'uregion.cpp',
-  'usearch.cpp',
-  'uspoof.cpp',
-  'uspoof_build.cpp',
-  'uspoof_conf.cpp',
-  'uspoof_impl.cpp',
-  'utf16collationiterator.cpp',
-  'utf8collationiterator.cpp',
-  'utmscale.cpp',
-  'utrans.cpp',
-  'vtzone.cpp',
-  'vzone.cpp',
-  'windtfmt.cpp',
-  'winnmfmt.cpp',
-  'wintzimpl.cpp',
-  'zonemeta.cpp',
-  'zrule.cpp',
-  'ztrans.cpp',
-  #  'decimalformatpattern.cpp',
-  #  'ucurr.cpp',
-  #  'digitlst.cpp',
-  #  'uspoof_wsconf.cpp',
-  #  'decfmtst.cpp',
-  #  'locdspnm.cpp',
-  #  'identifier_info.cpp',
-  #  'filteredbrk.cpp',
-)
+fs = import('fs')
```

Comment:
the fs variable leaks from the main meson.build file. no need to specify it again.